Assets/HoloToolkit/Input/Tests/Scripts/SphereGlobalKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class SphereGlobalKeywords : MonoBehaviour, ISpeechHandler
+    {
+        private void Start()
+        {
+            InputManager.Instance.AddGlobalListener(gameObject);
+        }
+
+        public void OnSpeechKeywordRecognized(SpeechKeywordRecognizedEventData eventData)
+        {
+            switch (eventData.RecognizedText.ToLower())
+            {
+                case "reset all":
+                    foreach (Renderer renderer in GetComponentsInChildren<Renderer>())
+                    { 
+                        renderer.material.color = Color.gray;
```

Comment:
It's based on the existing [SphereKeywords.cs](https://github.com/Microsoft/HoloToolkit-Unity/blob/0397cfab806d3e56dd18dffa4b90f4b511493f97/Assets/HoloToolkit/Input/Tests/Scripts/SphereKeywords.cs). I just changed it to apply the same color to all the children renderers.
What do you suggest in this case?